### PR TITLE
Peripheral Cluster Defaults

### PR DESCRIPTION
**Pull request by burrbull**
Add register ~~`Defaults`~~ `RegisterProperties` for Peripheral and ClusterInfo.

https://www.keil.com/pack/doc/CMSIS/SVD/html/elem_peripherals.html

### CODE REVIEW

File: src/svd/clusterinfo.rs [hunk 1]
```diff
@@ -7,22 +7,18 @@ use types::Parse;
...
 pub struct ClusterInfo {
     pub name: String,
     pub description: String,
     pub header_struct_name: Option<String>,
     pub address_offset: u32,
-    pub size: Option<u32>,
-    pub access: Option<Access>,
-    pub reset_value: Option<u32>,
-    pub reset_mask: Option<u32>,
+    pub defaults: Defaults,
```
Comment (Disasm) [inline src/svd/clusterinfo.rs]: (as discussed earlier) I'd rather use `register_properties` or `default_register_properties` here to express the meaning of these "defaults". Note that this property set is called `registerPropertiesGroup` in SVD spec.

Comment (burrbull) [reply src/svd/clusterinfo.rs]: Done

**Comment by burrbull** [comment]
@ryankurte  I didn't try build with `--all-features`.

Can you view again?

**Comment by burrbull** [comment]
bors retry

**Comment by ryankurte** [comment]
bors retry

**Comment by burrbull** [comment]
Please, try again.

**Comment by ryankurte** [comment]
ahh the CI dance ^_^

bors retry

**Comment by burrbull** [comment]
Retry https://github.com/japaric/svd/pull/72 also, please.

**Pull request merged**